packages/app-mobile/components/voiceTyping/AudioRecordingBanner.tsx

```
@@ -127,42 +124,28 @@ const useAudioRecorder = (onFileSaved: OnFileSavedCallback, onDismiss: ()=> void
-			});
-			setRecordingState(RecorderState.Recording);
-			const recording = new Audio.Recording();
-			await recording.prepareToRecordAsync(recordingOptions());
-			recording.setOnRecordingStatusUpdate(status => {
-				setDuration(status.durationMillis);
-			});
-			recordingRef.current = recording;
-			await recording.startAsync();
+			
+			await recording.prepareToRecordAsync();
+            recordingRef.current = recording;
```

ESLint

[error] Expected indentation of 3 tabs but found 12 spaces.

(`@typescript-eslint/indent`)

---

[error] Expected indentation of 3 tabs but found 12 spaces.

(indent)